components/podIndicator/index.jsx

```diff
@@ -168,6 +178,32 @@ export default function PodIndicator() {
               </ListItemIcon>
               <ListItemText disableTypography primary="Change Pod" />
             </ListItem>
+            <Tooltip
+              PopperProps={{
+                disablePortal: true,
+              }}
+              open={tooltipOpen}
+              disableFocusListener
+              disableHoverListener
+              disableTouchListener
+              title="Copied"
+            >
+              <ListItem
+                button
+                key="copy-text"
+                onClick={handlePodCopyClick}
+                classes={{ root: bem("menuItem") }}
+                data-testid={TESTCAFE_ID_POD_INDICATOR_COPY}
+              >
+                <ListItemIcon classes={{ root: bem("itemIcon") }}>
+                  <Icons
```

Comment:
The icon is not displaying. It looks like in line 174 for the **Change Pod** item it uses `<i>` but here it's using `<Icons>`

![Inrupt_PodBrowser](https://user-images.githubusercontent.com/11729241/151208071-4725c251-4294-449e-ab21-70cb0a758565.png)

Reply:
Missed Kevin's comment but have added that feature! The icon is a new icon added to Prism and is dependent on us being able to use a later version of Prism which is what we are working on in APPS-1145. <i> tags are how we pull the icons from Material UI, which our version doesn't contain the "content-copy" icon that we want to use here. <Icons> tags are when we get the icon from Prism (example in line 112). Want to keep this PR open until PRISM gets updated (which has to be done anyways) instead of spending time trying to figure out why "content-copy" is not available to us in Material UI.